frontend/src/locales/en.yml

```diff
@@ -336,15 +336,15 @@ activity:
     reopened-issue: reopened issue
     resolved-issue: resolved issue
     canceled-issue: canceled issue
-    empty: ''
+    empty: ""
```

Comment:
Could you please remove those quotation changes?

@xiaoluoboding @LiuJi-Jim not sure why this still happens. I thought we already added the VS settings in the project

Reply:
Prettier config, `singleQuote: false`

Reply:
@tianzhou Prettier also can format yaml.

Reply:
@tianzhou  If vscode enable save with format, then prettier will format yaml file, but yaml file format task is not in git commit hooks.